<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Employe</class>
 <widget class="QWidget" name="Employe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1317</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:white
</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>580</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rechID">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>490</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher par ID</string>
   </property>
  </widget>
  <widget class="QTableView" name="tabMAT">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>611</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="modifierMat">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="supprimerMat">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>490</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Ajouter / Modidier un Employe(e)</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>33</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Id Employe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>73</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Poste</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idE">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nom">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="poste">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>151</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="statut_dem">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>statut_dem</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="afficherMar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Affichage</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajoutMat">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="recherch_id">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>510</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="idMAT_Delete">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="statistiquesButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>480</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Statistique</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportPdfButton">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>490</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Exporter en PDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="assignationAutoButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>90</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Assignation Auto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="congesButton">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>170</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestion des Cong√©s</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
